portfolio.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: 'Inter';
+  src: url(./assets/fonts);
+}
+
+.header-welcome {
+  padding-top: 39px;
+  color: #28352f;
+  padding-left: 24px;
+}
+
+.menu-icon {
+  position: absolute;
+  width: 18px;
+  height: 20px;
+  bottom: 6641px;
+  right: 35px;
+  top: 62px;
+  left: 322px;
+}
```

Comment:
- [ ] Kindly, instead of given the position to this element, you could use `display flex` on order to position the elements inside your `nav` tag. In this way, you will be able to make it responsive on different sizes until the desktop version (768px) 💯